Live/Classes/Item.h

```diff
@@ -0,0 +1,18 @@
+#ifndef __ITEM__H__
+#define __ITEM__H__
+
+#include "GameObject.h"
+
+class Player;
```

Comment:
Don't forward declare, just make sure you are using guards/pragma once correctly

Reply:
Hm, how do I handle Player -> Inventory -> Item -> Player without forward declaring? I don't know enough c lol.